### ISSUE TITLE
Flaky test - TestPeerExchange

### ISSUE THREAD

**Issue by qfrank** [status-im/status-go#4835]
```
    waku_test.go:245: 
        	Error Trace:	/.../go/src/github.com/status-im/status-go/wakuv2/waku_test.go:245
        	Error:      	Received unexpected error:
        	            	bad bootstrap node "enr:-JG4QEUAaeYhKzNnlu-Xdq4pzEIB6eTdCJ16YZJBSW4cJpRvEFhfrtAFxT_6yjP1iIpjrDp_52uB6341vgvlLwWhjvSGAY3vDLyHgmlkgnY0gmlwhMCoAbyJc2VjcDI1NmsxoQNlt6CxE8F_9-sFyYsgmvS9zwhaOZnkweTTqJOj_ODQooN0Y3CC1GGFd2FrdTIJ": missing UDP port
        	Test:       	TestPeerExchange

    waku_test.go:275: 
        	Error Trace:	/.../go/src/github.com/status-im/status-go/wakuv2/waku_test.go:275
        	Error:      	Received unexpected error:
        	            	no peers discovered
        	Test:       	TestPeerExchange
```

can be reproduced with latest `develop` branch locally

**Comment by qfrank** [comment]
The first error get fixed by this [PR](https://github.com/status-im/status-go/pull/4851) , the later one can be only reproduced with another [PR](https://github.com/status-im/status-go/pull/4826) , the line `w.node.Host().Network().ClosePeer(peerInfo.ID)` within `identifyAndConnect` is the cause. Will be fixed in #4826

**Comment by qfrank** [comment]
fixed by relate PRs
